2.4/root/usr/share/container-scripts/mongodb/common.sh

```diff
@@ -152,7 +148,11 @@ function mongo_primary_member_addr() {
 # mongo_remove removes the current MongoDB from the cluster
 function mongo_remove() {
   local primary_addr
-  primary_addr="$(mongo_primary_member_addr)"
+  # we're ignoring exit code because without it, next if never gets executed
```

Comment:
As we talked on IRC, this comment is not very helpful because the next if is being introduced in this same patch.
AFAIU the reason we're doing this is to circumvent `set -o errexit`, ignore the error and allow the caller to proceed as if nothing has happened.

A concern I have is that whatever code calls `mongo_remove` or `mongo_add` cannot tell if it effectively removed or added a node from/to the replica set, or if the function terminated earlier.

It's more like they should be called `maybe_remove_from_replica_set` and `maybe_add_to_replica_set`.